### ISSUE TITLE
De-En 2020-05 Wikipedia

### ISSUE THREAD

**Issue by Ope30** [gnosygnu/xowa#717]
Sup @gnosygnu, Ope30 here!

You (maybe) missed my last post I posted a while ago in which I said that the new XOWA build unfortunately didn't fix these bugs for me. But honestly, I've been using XOWA so much in the last few weeks that I (personally) no longer care about them. It‘s alright. I am used to it. Everything else works fine. Either way, I would still download a new version. Always.

The latest builds are 3 months old. What is your opinion on creating and publishing a new build, German and English 2020-05 Wikipedia? Decide for yourself if you have the free time to do this. If necessary, we can decide on 2020-06. 👍🏼

**Comment by gnosygnu** [comment]
Hey. Good to hear from you again.

> You (maybe) missed my last post I posted a while ago in which I said that the new XOWA build unfortunately didn't fix these bugs for me

Yeah, sorry, missed that. 

> The latest builds are 3 months old. What is your opinion on creating and publishing a new build, German and English 2020-05 Wikipedia? Decide for yourself if you have the free time to do this. If necessary, we can decide on 2020-06. 👍🏼

I can try a German build, though not certain if 2020-05 or 2020-06. In brief, which is the worst bug that you're seeing in German? I can try to fix that before the next build

Other than that, hope everything is good for you in these times. Take care!

**Comment by Ope30** [comment]
> In brief, which is the worst bug that you're seeing in German? I can try to fix that before the next build

The worst bug for me personally is this one below, because it makes it hard to read the text the bug hides. Only city articles are affected because some of them, I think, contain Bundestag elections. The rest is at https://github.com/gnosygnu/xowa/issues/680 (JFYI)


![](https://user-images.githubusercontent.com/19750226/76729150-c0f0a200-6758-11ea-806e-7bb9cd4a3d16.png)


> I can try a German build, though not certain if 2020-05 or 2020-06.

Your choice. If you want to choose 2020-05, then do it. If you decide on 2020-06, then do it. You're the boss!



> Other than that, hope everything is good for you in these times. Take care!

You too, Gnosy. I wish you and your relatives the best of health!

**Comment by gnosygnu** [comment]
> The worst bug for me personally is this one below, because it makes it hard to read the text the bug hides. Only city articles are affected because some of them, I think, contain Bundestag elections. The rest is at #680 (JFYI)

Hmm, so this looks like the issue created here: #694

I know you said that you tried it...

> You (maybe) missed my last post I posted a while ago in which I said that the new XOWA build unfortunately didn't fix these bugs for me.

But just to be sure, did you actually download the version from here? https://github.com/gnosygnu/xowa/issues/694#issuecomment-605512119

desb42@ also said he couldn't reproduce, so I'm a bit puzzled.

Can you try the steps in the **Repro** section, and leave a comment there? Will track progress accordingly.

Thanks!

**Comment by Ope30** [comment]
> But just to be sure, did you actually download the version from here? #694 (comment)

Yep, I‘m 100% sure. I‘m using XOWA 64 bit 4.6.6.2003. The .exe file says it too.


> desb42@ also said he couldn't reproduce, so I'm a bit puzzled.

Me too. I don't understand why you two don't have the same problems I have. All I can think of is that I have an old version of Java installed on my computer, and downloading and installing the latest version might fix all that. I‘m not sure. Could this have something to do with it?


> Can you try the steps in the Repro section, and leave a comment there? Will track progress accordingly.

Sure. Will leave a post there

**Comment by Atlanta11** [comment]
Hmm, is it possible to download the English 2020-05 Wikipedia?

**Comment by desb42** [comment]
To throw in my two cents (pennies)

Having taken a longer look at `de.wikipedia.org/wiki/Bundestagswahl_2017` and the image posted by @Ope30, I note that xowa-gui menus are in German not English

This led me to the thought that the German locale may be affecting things (as both @gnosygnu  and I use English/American)
The expression errors being seen 'could' be due to decimal numbers being treated incorrectly

That is, the decimal point `.` is the character `,` in German
I wonder if this is causing the `#expr` logic some problems
The error `Expression error: Unrecognised word ","` could occur if a number, say 41.5, is treated as 41,5

**Comment by desb42** [comment]
Just to follow up on my previous comment
If I change the line

    | ERGEBNIS1    = 32.9<!--bit ...
to

    | ERGEBNIS1    = 32,9<!--bit ...
That is change `.` to `,`
I get
![germanerr](https://user-images.githubusercontent.com/861631/81475661-7c87fc80-9205-11ea-99e7-638c9c540436.jpg)
Which looks remarkably like @Ope30's screenshot

**Comment by gnosygnu** [comment]
@desb42 : Great find! I can reproduce something similar by setting my Windows Region to German

I think Luaj is probably using the user's region, whereas Lua is not. Let me dig through the code

Thanks!
